napari/layers/base/base.py

```diff
@@ -1625,7 +1626,7 @@ def _get_ray_intersections(
         dims_displayed: List[int],
         bounding_box: npt.NDArray,
         world: bool = True,
-    ) -> Union[Tuple[np.ndarray, np.ndarray], Tuple[None, None]]:
+    ) -> Tuple[Optional[np.ndarray], Optional[np.ndarray]]:
```

Comment:
Is that True ? Because this new one implies that `(None, ndarray)` is a value that can be returned, while before, it was not the case.

Reply:
Maybe for later in another PR, the docs should update the returns section to say it can be None.

Reply:
From what I can tell `intersect_line_with_axis_aligned_bounding_box_3d` cannot return None, so either both are None, or only one is None.